root/cdtoc/attach_list.tt

```diff
@@ -43,6 +49,13 @@
       </tr>
     </tr>
   [% END %]
+  [% UNLESS attachable_mediums.size %]
+    <tr[% ' class="even"' IF zebra % 2 == 0 %]>
+      <td class="error" colspan="7">
+        [% l('None of the mediums on this release can have the given CD TOC attached; either they have the wrong number of tracks, or the formats don’t support disc IDs.') %]
```

Comment:
This doesn't actually check for wrong format though. If you pick an MBID for a release with the wrong format it will load it without the error, it'll just be unselectable:

![Screenshot from 2020-02-22 19-32-51](https://user-images.githubusercontent.com/1069224/75096623-28785f00-55aa-11ea-82e0-40e4a0d40cba.png)

Reply:
I changed the message to not mention formats, and also only show this when it's an MBID search now.